<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RepeatedPlay</class>
 <widget class="QMainWindow" name="RepeatedPlay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>801</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RepeatedPlay</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="start_button">
        <property name="text">
         <string>Start Competition</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="infinite_btn">
        <property name="text">
         <string>Infinite Play</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListWidget" name="results_list"/>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="text_edit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>586</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
